src/net_processing.cpp

```diff
@@ -582,6 +582,19 @@ class PeerManagerImpl final : public PeerManager
      */
     bool MaybeDiscourageAndDisconnect(CNode& pnode, Peer& peer);
 
+    /** Handle a transaction whose result was not MempoolAcceptResult::ResultType::VALID.
+     * @param[in]   was_rejected    Whether this tx has already been rejected before and is being
```

Comment:
Maybe `was_orphan` would be a better name?

Reply:
I was calling it this, but in the future it could also be `was_failed_for_too_low_fee`.

Reply:
Ok, how about `add_to_extra_compact_txs`?

Reply:
Ok :handshake: `maybe_add_extra_compact_tx`